pkg/config/server_config.go

```diff
@@ -41,10 +40,14 @@ type ServerConfig struct {
 	SessionCookieDuration time.Duration `env:"SESSION_DURATION,default=24h"`
 	RevokeCheckPeriod     time.Duration `env:"REVOKE_CHECK_DURATION,default=5m"`
 
-	// CSRF Secret Key. Must be 32-bytes. Can be generated with tools/gen-secret
-	// Use the syntax of secret:// to pull the secret from secret manager.
-	// We assume the secret itself is base64 encoded. Use CSRFKey() to transform to bytes.
-	CSRFAuthKey string `env:"CSRF_AUTH_KEY,required"`
+	// CookieKeys is a slice of bytes. The odd values are hash keys to HMAC the
+	// cookies. The even values are block keys to encrypt the cookie. Both keys
+	// should be 64 bytes. The value's should be specified as base64 encoded.
+	CookieKeys Base64ByteSlice `env:"COOKIE_KEYS,required"`
```

Comment:
nice 👍

Reply:
We have to pull in the changes after https://github.com/google/exposure-notifications-server/pull/788#event-3608455621 for it to work with secrets thought. It'll work with local keys just fine.

Reply:
https://github.com/google/exposure-notifications-verification-server/pull/159